docs/self-managed/operational-guides/configure-flow-control/configure-flow-control.md

```diff
@@ -0,0 +1,311 @@
+  broker:
+    flowControl:
+      write:
+        enabled: false
+        rampUp: 0
+        limit: 1000
+        throttling:
+          enabled: false
+          acceptableBacklog: 100000
+          minimumLimit: 100
+          resolution: 15s
+```
+
+The zeebe engine can use a static rate limit to restrict the number of
+records that can be written per second, and it can use a dynamic throttling
+which changes the write rate based on the exporting backlog.
+
+The default values can be found in the [Zeebe broker standalone
+configuration template](https://github.com/camunda/camunda/blob/main/dist/src/main/config/broker.standalone.yaml.template) or in the [Zeebe broker configuration template](https://github.com/camunda/camunda/blob/main/dist/src/main/config/broker.yaml.template) in
+the `# flowControl` section.
+
+The `rampUp` value refers to the warmup period or how long it takes to go
+for the current write rate to the one set in the limit (assuming the rate
+higher than the limit). This value is given in seconds and can't be set
+null nor negative.
+
+The `limit` is the static value that we can configure to the write rate,
+therefore this can't be either null nor negative.
+
+The `throttling`, if enabled will additionally limit the write rate based on
+the exporting backlog.
+
+The exporting backlog is the quantity of records that were already
+processed but not yet exported. The increase of this is usually due to a
+mismatch of the rate of processed and exported records, or to exporting
+capability being degraded.
+
+The `throttling` algorithm used takes into account the ratio
+between the acceptable backlog and the actual backlog. If the acceptable
+backlog is at least twice as long as the real backlog we set the current
+rate as the limit (since we still have quite a margin, then we can increase
+the current processing rate to the defined limit). If the ratio between these
+is smaller than 2, then we calculate the rate by multiplying the ratio with
+the current exporting rate, with `minimumLimit` being our floor and `limit`
+being our ceiling. The intention is to increase or decrease proportionally the rate
+in between the defined limits according to the ratio of the acceptable
+backlog and the actual backlog, therefore if the acceptable backlog is
+larger than the backlog, we increase the rate, and we do the opposite if the
+backlog is larger than the acceptable backlog. From this we should expect to see the
+backlog stabilize around the acceptable backlog size (this only if the processing speed
+continues to outpace the exporting speed).
+
+The `resolution` value refers to the frequency that we adjust the
+throttling, and is given in seconds. Adjusting this value is useful to set
+the speed to which the processing rate can respond to changes.
+
+The current exporting rate is calculated by observing the number of records
+that we export during a period of time that's at least as long as the
+resolution window divided by the elapsed time.
```

Comment:
:wrench: I don't really understand this sentence. The exporting rate is the number of exported records per second, [averaged out over the last 5 minutes](https://github.com/camunda/camunda/blob/511100373c6d9d800bbad6c99ed4c96c73e29f89/zeebe/logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/flowcontrol/FlowControl.java#L80-L82).

Reply:
True, this sentence was not very clear. I applied the suggested explanation, thanks 🙌